extensions/interactions/base.py

```diff
@@ -89,6 +89,11 @@ class BaseInteraction(object):
     # containing this interaction. These should correspond to names of files in
     # feconf.DEPENDENCIES_TEMPLATES_DIR. Overridden in subclasses.
     _dependency_ids = []
+    # Additional JS library dependencies that should be loaded in pages
```

Comment:
So we have 2 kind of dependencies here. The original ones are needed for proper working of interaction itself and the new ones, which I am adding here, are required for prediction service associated with interaction. So should they be stored in a separate folder or in same folder? That's what I am asking. A separate location for prediction and original dependencies or not?